### PR TITLE
Adding support for listing Cosmos DB operations

### PR DESCRIPTION
**Pull request by dmakwana**
This checklist is used to make sure that common issues in a pull request are addressed. This will expedite the process of getting your pull request merged and avoid extra work on your part to fix issues discovered during the review process. 

### PR information
- [X] The title of the PR is clear and informative.
- [X] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For information on cleaning up the commits in your pull request, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).
- [X] Except for special cases involving multiple contributors, the PR is started from a fork of the main repository, not a branch.
- [X] If applicable, the PR references the bug/issue that it fixes.
- [X] Swagger files are correctly named (e.g. the `api-version` in the path should match the `api-version` in the spec).

### Quality of Swagger
- [X] I have read the [contribution guidelines](https://github.com/Azure/azure-rest-api-specs/blob/master/.github/CONTRIBUTING.md).
- [X] My spec meets the review criteria:
  - [X] The spec conforms to the [Swagger 2.0 specification](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md).
  - [X] The spec follows the guidelines described in the [Swagger checklist](../documentation/swagger-checklist.md) document.
  - [X] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.

### CODE REVIEW

**Comment by balajikris** [comment]
Looking at this..

File: specification/cosmos-db/resource-manager/Microsoft.DocumentDB/2015-04-08/cosmos-db.json [hunk 1]
```diff
@@ -851,6 +874,53 @@
...
+    },
+    "OperationListResult": {
+      "description": "Result of the request to list Resource Provider operations. It contains a list of operations and a URL link to get the next set of results.",
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/Operation"
+          },
+          "description": "List of Resource Provider operations supported by the Resource Provider resource provider."
```
Comment (balajikris) [inline specification/cosmos-db/resource-manager/Microsoft.DocumentDB/2015-04-08/cosmos-db.json]: please rewrite this description. Is this a  typo "Resource Provider resource provider" (or if you meant to convey something else, its not clear). Also, "list of operations supported by RP" sounds better than "list of RP operations supported by RP".

**Review by balajikris — COMMENTED** [review]
LGTM overall, but there is a nit-pick issue with the description given for OperationsList, please fix that.

**Comment by balajikris** [comment]
One more thing, it seems there is a syntax error with the spec and none of the CI jobs were able to proceed. Please fix this and make sure all documents have well formed json.

See these:
[1](https://travis-ci.org/Azure/azure-rest-api-specs/jobs/265322158)
[2](https://travis-ci.org/Azure/azure-rest-api-specs/jobs/265322160)
[3](https://travis-ci.org/Azure/azure-rest-api-specs/jobs/265322160)

**Review by balajikris — CHANGES_REQUESTED** [review]
Please respond to above feedback

**Comment by dmakwana** [comment]
@balajikris applied the fixes

**Review by balajikris — APPROVED** [review]

**Comment by balajikris** [comment]
Thanks @dmakwana  -- Merging.

**Pull request merged**